triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerRunnerImpl.scala

```diff
@@ -82,31 +82,30 @@ object TriggerRunnerImpl {
         Behaviors.receiveMessagePartial[Message] {
           case QueryACSFailed(cause) =>
             if (wasStopped) {
```

Comment:
Maybe there's an opportunity to refactor here as the behaviour when `wasStopped` is the basically the same for 2/3 messages, i.e. you could check `wasStopped` before the message. Not for this PR though.